### PR TITLE
phpunit/phpunit:^10.5.38, laminas/laminas-coding-standard:^3.0, Apply…

### PR DESCRIPTION
**Pull request by fezfez**

### CODE REVIEW

**Comment by fezfez** [comment]
@gsteel this should be good to go

File: test/Client/UseCaseTest.php [hunk 1]
```diff
@@ -30,28 +30,24 @@ class UseCaseTest extends TestCase
      *
      * @var string
      */
-    protected $baseuri;
+    protected string|array|false $baseuri;
```
Comment (gsteel) [inline test/Client/UseCaseTest.php]:
```suggestion
    private string $baseuri;
```

File: test/Client/UseCaseTest.php [hunk 2]
```diff
@@ -30,28 +30,24 @@ class UseCaseTest extends TestCase
...
 
     /**
      * Common HTTP client
      *
      * @var HTTPClient
      */
-    protected $client;
+    protected HTTPClient|null $client;
```
Comment (gsteel) [inline test/Client/UseCaseTest.php]:
```suggestion
    private HTTPClient $client;
```

Comment (gsteel) [reply test/Client/UseCaseTest.php]: Just delete the `tearDown` method

**Review by gsteel — APPROVED** [review]
Nice! Thanks for all your work @fezfez 

There's a comment on one test case that you can address if you like

**Comment by fezfez** [comment]
@gsteel done

**Review by gsteel — APPROVED** [review]
Thanks again @fezfez :)

**Pull request merged**